frontend/src/pages/mainPage/types/stat.ts

```diff
@@ -0,0 +1,34 @@
+import TimeLineIcon from '@/assets/icon/timeline.svg?react';
+import PlusIcon from '@/assets/icon/plus.svg?react';
+import BattleIcon from '@/assets/icon/battle.svg?react';
+
+export type StatType = 'TOTAL_BATTLES' | 'LIVE_BATTLES' | 'TOTAL_USERS';
+
+export const STAT_CONFIG: Record<
+  StatType,
+  {
+    label: string;
+    color: string;
+    background: string;
+    icon: React.FC<React.SVGProps<SVGSVGElement>>;
```

Comment:
현재 React.FC는 `현재 불필요한 추상화`, `암묵적인 children 타입 제공` 등으로 **안티패턴**으로 여겨지고 있는데, 작동에는 문제가 없어도 그래도 명시적 함수 타입을 적용한 방식이 좋을 듯합니다. 
```suggestion
icon: React.ComponentType<React.SVGProps<SVGSVGElement>>
```